package.json

```diff
@@ -7,8 +7,8 @@
     "url": "https://github.com/internetarchive/FirefoxNoMore404s/issues"
   },
   "devDependencies": {
-    "eslint": "3.1.1",
-    "web-ext": "1.3.0"
+    "eslint": "3.2.2",
+    "web-ext": "1.4.0"
```

Comment:
**web-ext@1.4.0** now let's us use `web-ext lint` directly to lint our web extension without needing to do any addons-linter tricks in .travis.yml.